textext.py

```diff
@@ -307,6 +307,9 @@ def do_convert(self, text, preamble_file, scale_factor, converter_class, old_nod
             return
 
         # -- Store textext attributes
+        new_node.attrib['{%s}textextver' % TEXTEXT_NS] = __version__.encode('string-escape')
```

Comment:
Call it "version" instead of "textextver". The namespace is already about textext.

Reply:
The idea to store these attributes is great, btw.

Reply:
Right. Changed it.